bin/beaver-install

```diff
@@ -7,13 +7,35 @@ set -eu
 
 d="$(dirname "$0")"
 
+# Copy a file only if it exist
+maybe_cp()
+{
+    if test -f "$1"
+    then
+        cp -v "$1" "$2"
+    fi
+}
+
+# Look if a particular context was given and copy the .dockerfile from it
+if test -n "${BEAVER_DOCKER_CONTEXT:-}"
+then
+    maybe_cp "$BEAVER_DOCKER_CONTEXT/.dockerignore" .dockerignore
+    maybe_cp "$BEAVER_DOCKER_CONTEXT/dockerignore" .dockerignore
+    docker_dir="-d $BEAVER_DOCKER_CONTEXT"
+else
+    BEAVER_DOCKER_CONTEXT=.
+    docker_dir=
+fi
+
 if test -r "Dockerfile${DIST:+.$DIST}"
```

Comment:
Missing `BEAVER_DOCKER_CONTEXT`

Reply:
Added a fixup commit for this. Good catch!

Reply:
(I was about to add a test for that before and said "meh, who uses that anyway :P")